docs/hazmat/backends/openssl.rst

```diff
@@ -125,6 +125,62 @@ seeded from the same pool as ``/dev/random``.
 +------------------------------------------+------------------------------+
 
 
+FIPS mode
+---------
+
+Federal Information Processing Standards is a public standard for use by
+US American government agencies. OpenSSL in FIPS mode prohibits the use of
+non-approved algorithms and parameters.
```

Comment:
This should a) indicate it means FIPS-140, there's a lot of other FIPS we don't care about.

Also I want it to say that we think FIPS is garbage, and you shouldn't use it unless you absolutely have to.

Reply:
I'm tempted to say we shouldn't have a FIPS section at all. If you need FIPS you have a special set of nonsense requirements and what version you're bound by and what matters is ultimately between you and the auditor. I'd rather say nothing right now and maybe later have some limited statements about how we handle FIPS. What do you think?

Reply:
That's fine with me, although just to be clear, most federal government contexts in which FIPS wil be (nominally) required don't actually have an auditor. They've got a person responsible for rubber stamping your paperwork, but no one will ever log into a computer or read a line of course code.